src/blocks/homepage-articles/view.php

```diff
@@ -46,6 +46,8 @@ function newspack_blocks_render_block_homepage_articles( $attributes ) {
 
 	$classes = Newspack_Blocks::block_classes( 'homepage-articles', $attributes );
```

Comment:
`block_classes` takes a third parameter called `$extras` which is an array of other classes to add. Therefore you could add `wpnbha` here without the extra line. 

```suggestion
	$classes = Newspack_Blocks::block_classes( 'homepage-articles', $attributes, array( 'wpnbha' ) );
```

Reply:
Ah, I didn't realize that! Updated in eb26d47  :)